include/gatekeeper_net.h

```diff
@@ -497,6 +500,22 @@ hw_filter_eth_available(const struct gatekeeper_if *iface)
 	return iface->hw_filter_eth && iface->rss;
 }
 
+static inline void
+set_ipv4_checksum(struct gatekeeper_if *iface, struct rte_mbuf *pkt,
+	struct rte_ipv4_hdr *ipv4)
+{
+	/*
+	 * The IP header checksum filed must be set to 0
+	 * in order to offload the checksum calculation.
```

Comment:
`rte_ipv4_cksum()` also requires the IP header checksum filed must be set to 0: https://doc.dpdk.org/api/rte__ip_8h.html#a8be1ccea98d6afa79fe0c8531eb266f7

Reply:
The idea is to just update the comment, not to remove it.